### PR TITLE
Webrtc media

### PR DESCRIPTION
**Pull request by hjain-git**
merging webrtc_media to main

### CODE REVIEW

File: bstack-webrtc-media/src/main/java/WebRTCTestRunner.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                bsLocalArgs.put("f", Config.getLocalFolderPath());
+                bsLocal.start(bsLocalArgs);
+            }
+            if (deviceType.equalsIgnoreCase("browser")) {
+                driver = new RemoteWebDriver(new URL(HUB_URL), options);
+                driver.manage().window().maximize();
+
+                if (userSelection.equalsIgnoreCase("1.3") || userSelection.equalsIgnoreCase("3.3")) {
+
+                    driver.get("http://" + System.getenv("BROWSERSTACK_USERNAME") +
```
Comment (samirans89) [inline bstack-webrtc-media/src/main/java/WebRTCTestRunner.java:52]: https

File: bstack-webrtc-media/src/main/java/WebRTCConf.java [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+
+                    System.out.println("Executing tests on Android Browser");
+                    for (int i = 0; i < AndroidSelections.length; i++) {
+                        // Creating capabilities for Android Mobile Browser.
+                        MutableCapabilities AndroidConfiguration = getAndroidConfiguration(AndroidSelections[i]);
+
+                        // Creating a session on Android Mobile Browser.
+                        createSession(AndroidConfiguration, "android", AndroidSelections[i]);
+                    }
+                    Thread.sleep(30000);
```
Comment (samirans89) [inline bstack-webrtc-media/src/main/java/WebRTCConf.java]: common thread.sleep at the end. see if we can reduce the value.

File: bstack-webrtc-media/src/main/java/WebRTCConf.java [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+            System.out.println("1 : Press 1 to initiate WebRTC test on Google Chrome Browser");
+            System.out.println("2 : Press 2 to initiate WebRTC test on Firefox Browser");
+            System.out.println("3 : Press 3 to initiate WebRTC test on Edge Browser");
+            System.out.println("4 : Press 4 to initiate WebRTC test on Safari Browser");
+            System.out.println("5 : Press 5 to initiate WebRTC test on Chrome Browser for Android ");
+            String userSelection = sc.next();
+
+            switch (userSelection) {
+                case "1":
+                    String[] selections = { "1.1", "1.2", "1.3" };
```
Comment (samirans89) [inline bstack-webrtc-media/src/main/java/WebRTCConf.java]: more intuitive names for the variables
e.g. Chrome_FakeMedia, Chrome_PreExistingMedia, Chrome_UserMedia

**Review by samirans89 — CHANGES_REQUESTED** [review]

File: bstack-webrtc-media/src/main/java/WebRTCTestRunner.java (hunk 1 above)
Comment (hjain-git) [reply bstack-webrtc-media/src/main/java/WebRTCTestRunner.java:52]: done

File: bstack-webrtc-media/src/main/java/WebRTCConf.java (hunk 2 above)
Comment (hjain-git) [reply bstack-webrtc-media/src/main/java/WebRTCConf.java]: done

File: bstack-webrtc-media/src/main/java/WebRTCConf.java (hunk 3 above)
Comment (hjain-git) [reply bstack-webrtc-media/src/main/java/WebRTCConf.java]: done

**Review by samirans89 — APPROVED** [review]

**Pull request merged**